### PR TITLE
python3Packages.docker: 4.4.0 -> 4.4.1

### PR DESCRIPTION
**Pull request by jonringer**
###### Motivation for this change
routine bump

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109067` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
30 packages marked as broken and skipped:
</summary>
<ul>
<li>cq-editor</li>
<li>mlflow-server</li>
<li>python37Packages.arviz</li>
<li>python37Packages.caffe</li>
<li>python37Packages.cnvkit</li>
<li>python37Packages.dftfit</li>
<li>python37Packages.flower</li>
<li>python37Packages.lammps-cython</li>
<li>python37Packages.mask-rcnn</li>
<li>python37Packages.mlflow</li>
<li>python37Packages.optuna</li>
<li>python37Packages.pygmt</li>
<li>python37Packages.pymc3</li>
<li>python37Packages.rl-coach</li>
<li>python37Packages.trackpy</li>
<li>python38Packages.arviz</li>
<li>python38Packages.caffe</li>
<li>python38Packages.cnvkit</li>
<li>python38Packages.dftfit</li>
<li>python38Packages.flower</li>
<li>python38Packages.lammps-cython</li>
<li>python38Packages.mask-rcnn</li>
<li>python38Packages.mlflow</li>
<li>python38Packages.optuna</li>
<li>python38Packages.pygmt</li>
<li>python38Packages.pymc3</li>
<li>python38Packages.rl-coach</li>
<li>python38Packages.trackpy</li>
<li>tebreak</li>
<li>truvari</li>
</ul>
</details>
<details>

  <summary>264 packages failed to build and are new build failures:</summary>
  <ul>
<li>apache-airflow: log was empty</li>
<li>(python37Packages.apache-airflow): log was empty</li>
<li>arion: log was empty</li>
<li>aws-sam-cli: log was empty</li>
<li>buildbot: log was empty</li>
<li>(python38Packages.buildbot): log was empty</li>
<li>buildbot-full: log was empty</li>
<li>(python38Packages.buildbot-full): log was empty</li>
<li>buildbot-ui: log was empty</li>
<li>(python38Packages.buildbot-ui): log was empty</li>
<li>csvs-to-sqlite: log was empty</li>
<li>(python38Packages.csvs-to-sqlite): log was empty</li>
<li>docker-compose: log was empty</li>
<li>fdroidserver: log was empty</li>
<li>python37Packages.Quandl: log was empty</li>
<li>(python37Packages.quandl): log was empty</li>
<li>python37Packages.acoustics: log was empty</li>
<li>python37Packages.altair: log was empty</li>
<li>python37Packages.aplpy: log was empty</li>
<li>python37Packages.apptools: log was empty</li>
<li>python37Packages.asdf: log was empty</li>
<li>python37Packages.atomman: log was empty</li>
<li>python37Packages.awkward0: log was empty</li>
<li>python37Packages.batchgenerators: log was empty</li>
<li>python37Packages.bkcharts: log was empty</li>
<li>python37Packages.buildbot: log was empty</li>
<li>python37Packages.buildbot-full: log was empty</li>
<li>python37Packages.buildbot-ui: log was empty</li>
<li>python37Packages.celery: log was empty</li>
<li>python37Packages.cirq: log was empty</li>
<li>python37Packages.clifford: log was empty</li>
<li>python37Packages.colorcet: log was empty</li>
<li>python37Packages.csvs-to-sqlite: log was empty</li>
<li>python37Packages.cufflinks: log was empty</li>
<li>python37Packages.dask: log was empty</li>
<li>python37Packages.dask-gateway: log was empty</li>
<li>python37Packages.dask-glm: log was empty</li>
<li>python37Packages.dask-image: log was empty</li>
<li>python37Packages.dask-jobqueue: log was empty</li>
<li>python37Packages.dask-ml: log was empty</li>
<li>python37Packages.dask-mpi: log was empty</li>
<li>python37Packages.dask-xgboost: log was empty</li>
<li>python37Packages.datasets: log was empty</li>
<li>python37Packages.datashader: log was empty</li>
<li>python37Packages.distributed: log was empty</li>
<li>python37Packages.django-raster: log was empty</li>
<li>python37Packages.djmail: log was empty</li>
<li>python37Packages.docker: log was empty</li>
<li>python37Packages.dockerspawner: log was empty</li>
<li>python37Packages.drms: log was empty</li>
<li>python37Packages.duckdb: log was empty</li>
<li>python37Packages.envisage: log was empty</li>
<li>python37Packages.fastparquet: log was empty</li>
<li>python37Packages.flammkuchen: log was empty</li>
<li>python37Packages.geopandas: log was empty</li>
<li>python37Packages.glymur: log was empty</li>
<li>python37Packages.google-cloud-automl: log was empty</li>
<li>python37Packages.google-cloud-bigquery: log was empty</li>
<li>python37Packages.graspologic: log was empty</li>
<li>python37Packages.holoviews: log was empty</li>
<li>python37Packages.hvplot: log was empty</li>
<li>python37Packages.ibis-framework: log was empty</li>
<li>python37Packages.image-match: log was empty</li>
<li>python37Packages.imagecorruptions: log was empty</li>
<li>python37Packages.imbalanced-learn: log was empty</li>
<li>python37Packages.intake: log was empty</li>
<li>python37Packages.ipydatawidgets: log was empty</li>
<li>python37Packages.jupyter-repo2docker: log was empty</li>
<li>python37Packages.mayavi: log was empty</li>
<li>python37Packages.mesa: log was empty</li>
<li>python37Packages.mlxtend: log was empty</li>
<li>python37Packages.moto: log was empty</li>
<li>python37Packages.nbsmoke: log was empty</li>
<li>python37Packages.nilearn: log was empty</li>
<li>python37Packages.osmnx: log was empty</li>
<li>python37Packages.osmpythontools: log was empty</li>
<li>python37Packages.pandas: log was empty</li>
<li>python37Packages.pandas-datareader: log was empty</li>
<li>python37Packages.partd: log was empty</li>
<li>python37Packages.phik: log was empty</li>
<li>python37Packages.pims: log was empty</li>
<li>python37Packages.ppscore: log was empty</li>
<li>python37Packages.pvlib: log was empty</li>
<li>python37Packages.pyarrow: log was empty</li>
<li>python37Packages.pybids: log was empty</li>
<li>python37Packages.pyfftw: log was empty</li>
<li>python37Packages.pymatgen: log was empty</li>
<li>python37Packages.pymatgen-lammps: log was empty</li>
<li>python37Packages.pyregion: log was empty</li>
<li>python37Packages.pytest-astropy: log was empty</li>
<li>python37Packages.pytest-astropy-header: log was empty</li>
<li>python37Packages.pytrends: log was empty</li>
<li>python37Packages.pywick: log was empty</li>
<li>python37Packages.qiskit: log was empty</li>
<li>python37Packages.qiskit-aer: log was empty</li>
<li>python37Packages.qiskit-aqua: log was empty</li>
<li>python37Packages.qiskit-ibmq-provider: log was empty</li>
<li>python37Packages.qiskit-ignis: log was empty</li>
<li>python37Packages.qiskit-terra: log was empty</li>
<li>python37Packages.radio_beam: log was empty</li>
<li>python37Packages.reproject: log was empty</li>
<li>python37Packages.rpy2: log was empty</li>
<li>python37Packages.runway-python: log was empty</li>
<li>python37Packages.scikit-bio: log was empty</li>
<li>python37Packages.scikitimage: log was empty</li>
<li>python37Packages.seaborn: log was empty</li>
<li>python37Packages.shap: log was empty</li>
<li>python37Packages.skorch: log was empty</li>
<li>python37Packages.slicer: log was empty</li>
<li>python37Packages.sparse: log was empty</li>
<li>python37Packages.spectral-cube: log was empty</li>
<li>python37Packages.spyder-kernels_0_5: log was empty</li>
<li>python37Packages.statsmodels: log was empty</li>
<li>python37Packages.streamz: log was empty</li>
<li>python37Packages.stumpy: log was empty</li>
<li>python37Packages.stytra: log was empty</li>
<li>python37Packages.sumo: log was empty</li>
<li>python37Packages.sunpy: log was empty</li>
<li>python37Packages.tablib: log was empty</li>
<li>python37Packages.tensorboardx: log was empty</li>
<li>python37Packages.tensorly: log was empty</li>
<li>python37Packages.test-tube: log was empty</li>
<li>python37Packages.tiledb: log was empty</li>
<li>python37Packages.traittypes: log was empty</li>
<li>python37Packages.transformers: log was empty</li>
<li>python37Packages.uproot3: log was empty</li>
<li>python37Packages.uproot3-methods: log was empty</li>
<li>python37Packages.vega: log was empty</li>
<li>python37Packages.vega_datasets: log was empty</li>
<li>python37Packages.vidstab: log was empty</li>
<li>python37Packages.wrf-python: log was empty</li>
<li>python37Packages.xarray: log was empty</li>
<li>python37Packages.xgboost: log was empty</li>
<li>python37Packages.yfinance: log was empty</li>
<li>python38Packages.Quandl: log was empty</li>
<li>(python38Packages.quandl): log was empty</li>
<li>python38Packages.acoustics: log was empty</li>
<li>python38Packages.altair: log was empty</li>
<li>python38Packages.aplpy: log was empty</li>
<li>python38Packages.apptools: log was empty</li>
<li>python38Packages.asdf: log was empty</li>
<li>python38Packages.atomman: log was empty</li>
<li>python38Packages.awkward0: log was empty</li>
<li>python38Packages.batchgenerators: log was empty</li>
<li>python38Packages.bkcharts: log was empty</li>
<li>python38Packages.celery: log was empty</li>
<li>python38Packages.cirq: log was empty</li>
<li>python38Packages.clifford: log was empty</li>
<li>python38Packages.colorcet: log was empty</li>
<li>python38Packages.cufflinks: log was empty</li>
<li>python38Packages.dask: log was empty</li>
<li>python38Packages.dask-gateway: log was empty</li>
<li>python38Packages.dask-glm: log was empty</li>
<li>python38Packages.dask-image: log was empty</li>
<li>python38Packages.dask-jobqueue: log was empty</li>
<li>python38Packages.dask-ml: log was empty</li>
<li>python38Packages.dask-mpi: log was empty</li>
<li>python38Packages.dask-xgboost: log was empty</li>
<li>python38Packages.datasets: log was empty</li>
<li>python38Packages.datashader: log was empty</li>
<li>python38Packages.distributed: log was empty</li>
<li>python38Packages.django-raster: log was empty</li>
<li>python38Packages.djmail: log was empty</li>
<li>python38Packages.docker: log <a href="https://termbin.com/bton">https://termbin.com/bton</a></li>
<li>python38Packages.dockerspawner: log was empty</li>
<li>python38Packages.drms: log was empty</li>
<li>python38Packages.duckdb: log was empty</li>
<li>python38Packages.envisage: log was empty</li>
<li>python38Packages.fastparquet: log was empty</li>
<li>python38Packages.flammkuchen: log was empty</li>
<li>python38Packages.geopandas: log was empty</li>
<li>python38Packages.glymur: log was empty</li>
<li>python38Packages.google-cloud-automl: log was empty</li>
<li>python38Packages.google-cloud-bigquery: log was empty</li>
<li>python38Packages.graspologic: log was empty</li>
<li>python38Packages.holoviews: log was empty</li>
<li>python38Packages.hvplot: log was empty</li>
<li>python38Packages.image-match: log was empty</li>
<li>python38Packages.imagecorruptions: log was empty</li>
<li>python38Packages.imbalanced-learn: log was empty</li>
<li>python38Packages.intake: log was empty</li>
<li>python38Packages.ipydatawidgets: log was empty</li>
<li>python38Packages.jupyter-repo2docker: log was empty</li>
<li>python38Packages.mayavi: log was empty</li>
<li>python38Packages.mesa: log was empty</li>
<li>python38Packages.mlxtend: log was empty</li>
<li>python38Packages.moto: log was empty</li>
<li>python38Packages.nbsmoke: log was empty</li>
<li>python38Packages.nilearn: log was empty</li>
<li>python38Packages.osmnx: log was empty</li>
<li>python38Packages.osmpythontools: log was empty</li>
<li>python38Packages.pandas: log was empty</li>
<li>python38Packages.pandas-datareader: log was empty</li>
<li>python38Packages.partd: log was empty</li>
<li>python38Packages.phik: log was empty</li>
<li>python38Packages.pims: log was empty</li>
<li>python38Packages.ppscore: log was empty</li>
<li>python38Packages.pvlib: log was empty</li>
<li>python38Packages.pyarrow: log was empty</li>
<li>python38Packages.pybids: log was empty</li>
<li>python38Packages.pyfftw: log was empty</li>
<li>python38Packages.pymatgen: log was empty</li>
<li>python38Packages.pymatgen-lammps: log was empty</li>
<li>python38Packages.pyregion: log was empty</li>
<li>python38Packages.pytest-astropy: log was empty</li>
<li>python38Packages.pytest-astropy-header: log was empty</li>
<li>python38Packages.pytrends: log was empty</li>
<li>python38Packages.pywick: log was empty</li>
<li>python38Packages.qiskit: log was empty</li>
<li>python38Packages.qiskit-aer: log was empty</li>
<li>python38Packages.qiskit-aqua: log was empty</li>
<li>python38Packages.qiskit-ibmq-provider: log was empty</li>
<li>python38Packages.qiskit-ignis: log was empty</li>
<li>python38Packages.qiskit-terra: log was empty</li>
<li>python38Packages.radio_beam: log was empty</li>
<li>python38Packages.reproject: log was empty</li>
<li>python38Packages.rpy2: log was empty</li>
<li>python38Packages.runway-python: log was empty</li>
<li>python38Packages.scikit-bio: log was empty</li>
<li>python38Packages.scikitimage: log was empty</li>
<li>python38Packages.seaborn: log was empty</li>
<li>python38Packages.shap: log was empty</li>
<li>python38Packages.skorch: log was empty</li>
<li>python38Packages.slicer: log was empty</li>
<li>python38Packages.sparse: log was empty</li>
<li>python38Packages.spectral-cube: log was empty</li>
<li>python38Packages.spyder-kernels_0_5: log was empty</li>
<li>python38Packages.statsmodels: log was empty</li>
<li>python38Packages.streamz: log was empty</li>
<li>python38Packages.stumpy: log was empty</li>
<li>python38Packages.stytra: log was empty</li>
<li>python38Packages.sumo: log was empty</li>
<li>python38Packages.sunpy: log was empty</li>
<li>python38Packages.tablib: log was empty</li>
<li>python38Packages.tensorboardx: log was empty</li>
<li>python38Packages.tensorly: log was empty</li>
<li>python38Packages.test-tube: log was empty</li>
<li>python38Packages.tiledb: log was empty</li>
<li>python38Packages.traittypes: log was empty</li>
<li>python38Packages.transformers: log was empty</li>
<li>python38Packages.uproot3: log was empty</li>
<li>python38Packages.uproot3-methods: log was empty</li>
<li>python38Packages.vega: log was empty</li>
<li>python38Packages.vega_datasets: log was empty</li>
<li>python38Packages.vidstab: log was empty</li>
<li>python38Packages.wrf-python: log was empty</li>
<li>python38Packages.xarray: log was empty</li>
<li>python38Packages.xgboost: log was empty</li>
<li>python38Packages.yfinance: log was empty</li>
<li>python39Packages.docker: log <a href="https://termbin.com/z25t">https://termbin.com/z25t</a></li>
<li>python39Packages.dockerspawner: log was empty</li>
<li>python39Packages.jupyter-repo2docker: log was empty</li>
<li>sourcehut.buildsrht: log was empty</li>
<li>sourcehut.dispatchsrht: log was empty</li>
<li>sourcehut.gitsrht: log was empty</li>
<li>sourcehut.hgsrht: log was empty</li>
<li>sourcehut.hubsrht: log was empty</li>
<li>sourcehut.listssrht: log was empty</li>
<li>sourcehut.mansrht: log was empty</li>
<li>sourcehut.metasrht: log was empty</li>
<li>sourcehut.pastesrht: log was empty</li>
<li>sourcehut.todosrht: log was empty</li>
<li>streamlit: log was empty</li>
<li>visidata: log was empty</li>
  </ul>
</details>
